lib/screens/home_calendar.dart

```diff
@@ -41,42 +39,252 @@ class _HomeCalendarState extends State<HomeCalendar> {
-                    ),
+            HomeHeaderExpanded(title: widget.title, actions: [
+              Container(
+                width: 56.0,
+                child: RawMaterialButton(
+                  padding: EdgeInsets.all(0.0),
+                  shape: CircleBorder(),
+                  onPressed: () =>
+                      General.instance.navigateScreenNamed(context, '/ledgers'),
+                  child: Icon(
+                    Icons.book,
+                    color: Colors.white,
                   ),
                 ),
-                Container(
-                  width: 56.0,
-                  child: RawMaterialButton(
-                    padding: EdgeInsets.all(0.0),
-                    shape: CircleBorder(),
-                    onPressed: () => General.instance.navigateScreenNamed(context, '/ledger_item_add'),
-                    child: Icon(
-                      Icons.add,
-                      color: Colors.white,
-                    ),
+              ),  
+              Container(
+                width: 56.0,
+                child: RawMaterialButton(
+                  padding: EdgeInsets.all(0.0),
+                  shape: CircleBorder(),
+                  onPressed: () => General.instance
+                      .navigateScreenNamed(context, '/ledger_item_add'),
+                  child: Icon(
+                    Icons.add,
+                    color: Colors.white,
                   ),
                 ),
-              ]
-            ),
+              ),
+            ]),
           ];
         },
         controller: _scrollController,
-        body: Center(
-          child: Text("Sample Text"),
+        body: Container(
+          child: new MyHomePage(),
         ),
       ),
     );
   }
 }
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key}) : super(key: key);
+
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  DateTime _currentDate2 = DateTime(2019, 2, 3);
+  String _currentMonth = '';
+
+  EventList<Event> _markedDateMap = new EventList<Event>();
+
+  CalendarCarousel _calendarCarouselNoHeader;
+
+  /// ledgerList from parents
+  List<LedgerItem> _ledgerList = new List<LedgerItem>();
+
+  /// for bottom list UI
+  List<LedgerItem> _ledgerListOfSelectedDate = new List<LedgerItem>();
+
+  void selectDate(
+    DateTime date,
+    List<Event> events,
+  ) {
+    setState(() {
+      _currentDate2 = date;
+      _ledgerListOfSelectedDate.clear();
+      _ledgerList.forEach((item) {
+        if (item.selectedDate == date) {
+          _ledgerListOfSelectedDate.add(item);
+        }
+      });
+    });
+    events.forEach((event) => print(event.title));
+    print(_ledgerListOfSelectedDate.length);
+  }
+
+  @override
+  void initState() {
+    _ledgerList.add(new LedgerItem(
+        price: -12000,
+        category: Category(iconId: 8, label: '운동', type: CategoryType.CONSUME),
```

Comment:
Could you localize this and other parameters with `Localization`? There are already a few sample codes in our project!

Reply:
such as `Localization.trans('EXCERSISE')` in `en.json` and `ko.json`.

Reply:
I've fixed code as you requested!